package.json

```diff
@@ -23,7 +23,6 @@
   },
   "dependencies": {
     "@opentelemetry/api": "^1.9.0",
-    "@opentelemetry/semantic-conventions": "^1.26.0",
```

Comment:
I'm removing this dependency, as it makes more sense for us to keep this in sync "in-house" within the SDK, considering `semantic-conventions` had marked all the interfaces we were using as deprecated.